phishtray/base.py

```diff
@@ -29,11 +29,16 @@ class Meta:
 class MultiTenantQuerySet(QuerySet):
 
     def filter_by_org_private(self, user):
+        User = apps.get_model('users', 'User')
+
         if not user or not isinstance(user, User):
             return self.none()
 
         if not user.is_superuser:
-            return self.filter(organization=user.organization)
+            if user.organization is None:
+                return self.none()
```

Comment:
Can't we add `if user.organization is None` to line 34? 
Is it different when an admin is not a superuser? Means when an admin has no organization yet, will he still have the access to this `MultiTenant` mixins?

Reply:
Regular admins should have organizations at all time, maybe we should reject them at login - but that's another story.
Returning none here when an admin has no organization is aiming to restrict the user to do anything.
Otherwise they could copy public exercises etc.